### ISSUE TITLE
508-defect-3 [COGNITION]: Links SHOULD be written as links OR buttons SHOULD be styled as buttons

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18714]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## User Story or Problem Statement

As a user, I expect to be able to open a link in a new tab so that I can navigate in a way that works best for me.

## Details

The "If you know VA Form 21-526EZ is right, apply now" link is written as a `button` which is materially dishonest. Users may become frustrated that they are unable to open a new tab. Screen reader users may be confused that the text seems more like a link despite being announced as a button.

For more on this long-tail effort, check out our [buttons vs. links Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1610052905994/253723ef4b2ee9a726f538876a257df94d468db1) and this [Slack thread discussing _why_ material honesty is so important](https://dsva.slack.com/archives/C8E985R32/p1610659601009200?thread_ts=1610652897.002600&cid=C8E985R32).

"A few years at an accessibility event, a blind man was telling a story about how he was on the phone with someone who was trying to help him sign up for a service. The person on the phone told him to click the “sign up button” and he couldn’t find it because it was a link. He told the story with a lot more gravity than I can, but that was the point where I realized how important the human-human communication part of accessibility is, it’s not just machine to human." - @timwright12 

## Acceptance Criteria

- [ ] "If you know VA Form 21-526EZ is right, apply now" is written with an `a` tag OR
- [ ] "If you know VA Form 21-526EZ is right, apply now" is styled as a button

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction` in browser
2. Right click on "If you know VA Form 21-526EZ is right, apply now"
3. Confirm you cannot open in a new tab (it's a button)

## Solution (if known)

Use `a` instead of `button` or style the `button` as a `button`

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [VA Design System guidance on buttons](https://design.va.gov/components/buttons#guidance)

## Screenshots or Trace Logs

<img width="1108" alt="Screen Shot 2021-01-19 at 10 14 40 AM" src="https://user-images.githubusercontent.com/14154792/105054411-f5e81800-5a3f-11eb-8069-85c61813f25a.png">

**Comment by Mottie** [comment]
Merged. This change will be available in staging shortly.

**Comment by joshkimux** [comment]
Confirmed this has been updated!! 💯
